frontend/src/component/project/Project/ProjectFeatureToggles/ProjectFeatureToggles.tsx

```diff
@@ -336,8 +372,13 @@ export const ProjectFeatureToggles = ({
                                 name: env,
                                 enabled:
                                     featureEnvironments?.find(
-                                        feature => feature?.name === env
+                                        fe => fe?.name === env
                                     )?.enabled || false,
+                                variantCount:
+                                    featureEnvironments?.find(
+                                        fe => fe.name === env
+                                    )?.variantCount || 0,
+                                // othersUseVariants: featureEnvironments?.find(fe => fe.variantCount > 0 && fe.enabled)?.enabled || false,
```

Comment:
Also @nunogois I don't understand this part... without adding `variantCount` here I don't see that field in another part of the code, but I tried using `othersUseVariants` but that's not visible... so I have the feeling that I'm doing something wrong

Reply:
Hmm if I understood correctly you probably want something like:

```TS
othersUseVariants:
  featureEnvironments?.some(
      environment =>
          environment.variantCount > 0 &&
          environment.enabled
  ) || false,
```

You can then add `othersUseVariants: boolean;` to `type ListItemType` on line 66.

Which means you could do something like this on the cell conditional rendering:

```TS
feature.environments[name]?.othersUseVariants &&
feature.environments[name]?.variantCount ===
    0 &&
feature.environments[name].enabled
```

You could probably also just compute whether to show the warning inside the memo, since it seems you have everything you need in there.

Reply:
Thanks! an awesome suggestion!